.pre-commit-config.yaml

```
@@ -12,6 +12,7 @@ repos:
         verbose: true
         stages: [commit-msg]
   - repo: https://github.com/charliermarsh/ruff-pre-commit
-    rev: v0.1.0
+    rev: v0.1.2
     hooks:
       - id: ruff
+        id: ruff-format
```

I'm pretty sure this needs to be: 

```suggestion
    hooks:
      - id: ruff
      - id: ruff-format
```
Without the `-` it's invalid yaml apparently.

---

Yep, you're right. Fixed.